pontoon/base/templates/translate.html

```diff
@@ -239,27 +239,31 @@ <h3 class="no-match"><div class="fa fa-exclamation-circle"></div>No results</h3>
         <li><a href="#plural6"><span></span><sup></sup></a></li>
       </ul>
     </nav>
-    <textarea dir="auto" lang="{{ locale.code }}" id="translation" placeholder="Type translation and press Enter to save"></textarea>
-
+    
+    <textarea {% if not user.is_authenticated() %} readonly {% endif %} dir="auto" lang="{{ locale.code }}" id="translation" placeholder="Type translation and press Enter to save"></textarea>
     <menu>
-      <div id="unsaved" class="warning-overlay">
-        <a href="#" class="cancel">&times;</a>
-        <h2>You have unsaved changes</h2>
-        <p>Sure you want to leave?</p>
-        <button id="leave-anyway">Leave anyway</button>
-      </div>
-      <div id="quality" class="warning-overlay">
-        <a href="#" class="cancel">&times;</a>
-        <h2>The following checks have failed</h2>
-        <ul></ul>
-        <button id="save-anyway"></button>
-      </div>
-      <div id="translation-length">
-        <span class="current-length"></span>|<span class="original-length"></span>
-      </div>
-      <button id="copy" title="Copy From Source (Alt + C)">Copy</button>
-      <button id="clear" title="Clear Translation (Alt + Backspace)">Clear</button>
-      <button id="save" title="Save Translation (Enter)"></button>
+      {% if user.is_authenticated() %}
+        <div id="unsaved" class="warning-overlay">
+          <a href="#" class="cancel">&times;</a>
+          <h2>You have unsaved changes</h2>
+          <p>Sure you want to leave?</p>
+          <button id="leave-anyway">Leave anyway</button>
+        </div>
+        <div id="quality" class="warning-overlay">
+          <a href="#" class="cancel">&times;</a>
+          <h2>The following checks have failed</h2>
+          <ul></ul>
+          <button id="save-anyway"></button>
+        </div>
+        <div id="translation-length">
+          <span class="current-length"></span>|<span class="original-length"></span>
+        </div>
+        <button id="copy" title="Copy From Source (Alt + C)">Copy</button>
+        <button id="clear" title="Clear Translation (Alt + Backspace)">Clear</button>
+        <button id="save" title="Save Translation (Enter)"></button>
+      {% else %}
+        <p id="sign-in-required">Sign in to suggest or make changes.</p>
```

Comment:
Let's simplify this to "Sign in to translate". Sorry for my split personality. ;)